react_app/src/App.scss

```diff
@@ -0,0 +1,183 @@
+.App {
+  font-family: sans-serif;
+  text-align: center;
+}
+
+html {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  -webkit-box-sizing: inherit;
+          box-sizing: inherit;
+}
+
+body {
+  outline : none;
+  margin: 20px 0;
+  padding: 0;
+  line-height: 1;
+  font-size: 16px;
+  font-family: 'Roboto', sans-serif;
+  color: #202020;
+  background-color: #fbfbfb;
+  font-smooth: always;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.container {
+  max-width: 1140px;
+  width: 100%;
+  margin: auto;
+}
+
+@mixin styling() {
+  background-color: white;
+  border-color: #ccc;
+  border-radius: 4px;
+  border-style: solid;
+  border-width: 1px;
+  box-shadow: 0 .125rem .25rem rgba(0,0,0,.075) !important;
+}
+
+.creation-modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width:100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.6);
+
+  .modal-main {
+    position:fixed;
+    background: white;
+    width: 80%;
+    height: 30%;
+    top:50%;
+    left:50%;
+    transform: translate(-50%,-50%);
+    border-radius : 10px;
+
+    .cancel {
+      border : none;
+      border-radius : 5px;
+      background-color: #e33440;
+      font-size : 18px;
+      color : #fbfbfb;
+      width : 20%;
+      height : 50px;
+      transition : background-color 0.2s ease-in-out;
+    }
+
+    .cancel:hover {
```

Comment:
you can/should use the sexy nested syntax here too

```
.class {
    &:hover {
        ...
    }
}
```

Reply:
Fixed!